camel/types/__init__.py

```diff
@@ -43,36 +43,44 @@
     NotGiven,
     ParsedChatCompletion,
 )
+from .openbb_types import (
+    OpenBBBaseProtocol,
+    OpenBBClient,
+    OpenBBType,
+)
 from .unified_model_type import UnifiedModelType
 
 __all__ = [
-    'RoleType',
-    'ModelType',
-    'TaskType',
-    'TerminationMode',
-    'OpenAIBackendRole',
-    'EmbeddingModelType',
-    'VectorDistance',
-    'StorageType',
-    'Choice',
```

Comment:
Hi Anjie，Please confirm whether any accidental deletions have occurred.

Reply:
No, I just re-ordered them for passing the ruff-format check.